indico/modules/events/export.yaml

```diff
@@ -221,6 +221,12 @@ export:
     fks_out:
       - event_role_id
 
+  MenuEntryPrincipal:
```

Comment:
> Also if you could provide some guidance on the exporters, I really don't know what I am doing there and if it is correct.

Since you asked about how the export stuff works some time ago: I'll add some comments below to explain it